### DISCUSSION TITLE
NOAA 'Omics Data Management Plans Issues/Questions?

### DISCUSSION THREAD

**Discussion by marinednadude** [NOAA-Omics/NOAA-Omics#16] [4-DataMgmt]
Has anyone run into issues surrounding the [NOAA 'Omics Data Management Plans](https://noaa-omics-dmg.readthedocs.io/en/latest/dmp-template.html)?

**Comment by ksilnoaa** [comment]
@marinednadude what types of issues?  We've received guidance that they will be accepted by OAR.

**Reply by marinednadude** [reply]
I think this comment arose from a meeting with Kelly Goodwin where she asked to start a thread for folks who may have come across issues around the data management plans and implementation. This might not be the best forum. 

So far we have not heard any concerns or issues.
